docs/pages/project/contributing/contributing-adapters.md

```diff
@@ -8,8 +8,6 @@ type: project
 category: contributing
 ---
 
-**See [Extensibility: Service Mesh Adapters]({{site.baseurl}}/extensibility/adapters)**
+Meshery Adapters are the Extension Points in Meshery's architecture. Their design, the process of creating a new adapter and guidelines for contributing to an existing adapter is documented in [Extensibility: Service Mesh](https://docs.meshery.io/extensibility/adapters).
```

Comment:
Add this complete -> Extensibility: Service Mesh Adapters

Reply:
Understood but could you please suggest why do we need to add the "see extensibility service mesh" at the initial.
Anyhow we are using it below at the end of the line right.
So I felt that it got repeated twice which is not needed.
Correct if i am wrong somewhere in this

Reply:
yes good point @measutosh  , remove that See line , we dont need that 
I had asked for the bottom line after 11 add Adapters Word

Reply:
Understood, made the suggested change

And i have given few comments on the other PR I raised which you closed, could you please check there once. 

Thanks